app/__init__.py

```diff
@@ -15,10 +15,24 @@
         'Synopsis':'Mario goes to save princess Peach'
     }
         ]
-
+#gets all books
 @app.route('/api/v1/books')
 def get_books():
     return jsonify({'books': books})
 
+#gets a single book by id
+@app.route('/api/v1/books/<int:book_id>', methods=['GET'])
```

Comment:
Add doc string explaining what the functionality needs to accomplish. Refer to this resource to understand docstrings http://www.pythonforbeginners.com/basics/python-docstrings